doc/source/operations.rst

```diff
@@ -589,3 +589,134 @@ To avoid having to pass "-n openstack" all the time:
 .. code-block:: console
 
    kubectl config set-context $(kubectl config current-context) --namespace=openstack
+
+Cheat-sheet
+===========
+
+**Kubernetes useful commands**
+
+To get namespaces
+
+.. code-block:: console
+
+   kubectl get namespaces
+
+To get the cluster nodes and their status
+
+.. code-block:: console
+
+  kubectl get nodes
+
+To list the pods are running in what node in an specific namespace
+
+.. code-block:: console
+
+   kubectl get pods -o wide -n openstack | grep Running
+
+To list the pods are running in what node of all namespaces
+
+.. code-block:: console
+
+   kubectl get pods -o wide --all-namespaces | grep Running
+
+To list the pods are NOT running in what node of all namespaces
+
+.. code-block:: console
+
+    kubectl get pods -o wide --all-namespaces | grep -v Running
+
+To list the pods with column customization. In the example only shows the name
+and the worker where is this pod more info at 
+  - https://kubernetes.io/docs/reference/kubectl/overview/#custom-columns
+  - https://kubernetes.io/docs/reference/kubectl/jsonpath/
+
+.. code-block:: console
+
+    kubectl -n openstack get pods -o=custom-columns=NAME:.metadata.name,WORKER:.spec.nodeName
+
+To get the logs of a pod
+
+.. code-block:: console
+
+ kubectl logs <pod-name>
```

Comment:
It would be good if you provide `-n ` option to most of the commands

Reply:
Done